code/game/objects/items/weapons/weaponry.dm

```diff
@@ -31,11 +31,100 @@
 	throw_range = 4
 	throwforce = 10
 	w_class = 1
+	var/reskinned = 0
 
 /obj/item/weapon/nullrod/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is impaling \himself with the [src.name]! It looks like \he's trying to commit suicide.</span>")
 	return (BRUTELOSS|FIRELOSS)
 
+/obj/item/weapon/nullrod/AltClick(mob/user)
+	..()
+	if(!user.canUseTopic(user))
+		user << "<span class='warning'>You can't do that right now!</span>"
+		return
+	if(!in_range(src, user))
+		return
+	if(reskinned)
+		return
+	reskin_holy_weapon(user)
+
+/obj/item/weapon/nullrod/proc/reskin_holy_weapon(mob/M)
+	var/choice = input(M,"What theme would you like for your holy weapon?","Holy Weapon Theme") as null|anything in list("null rod", "god hand", "red staff", "blue staff", "claymore", "dark blade", "sord", "scythe")
+
+	if(src && choice && !M.stat && in_range(M,src) && !M.restrained() && M.canmove && !reskinned)
+		switch(choice)
+			if("null rod")
+				M << "On second thought, the null rod suits you just fine."
+			if("god hand")
+				icon_state = "disintegrate"
+				item_state = "disintegrate"
+				name = "god hand"
+				desc = "This hand of yours glows with an aweseome power."
+				flags = ABSTRACT | NODROP
+				w_class = 5
+			if("red staff")
+				icon_state = "godstaff-red"
+				item_state = "godstaff-red"
+				name = "holy staff"
+				desc = "It has a mysterious, protective aura."
+				w_class = 5
+				force = 5
+				slot_flags = SLOT_BACK
+				block_chance = 50
+			if("red staff")

```

Comment:
Can you do if(redstaff || bluestaff) in a switch statement? 

I guess that would make sense but I've never done that/seen it done